README.md

```diff
@@ -5,171 +5,45 @@
-
-```ts
-// lib/prisma.ts 
-import { PrismaClient } from "@prisma/client"
-
-const globalForPrisma = global as unknown as { prisma: PrismaClient }
-    
-export const prisma = globalForPrisma.prisma || new PrismaClient()
-    
-if (process.env.NODE_ENV !== 'production') globalForPrisma.prisma = prisma
-    
-export default prisma
-```
-
-#### Importing the global Prisma Client instance
-We recommend importing this instance into the script tags of your `.vue` files like so: 
-
-```vue
-<script lang="ts" setup>
-// Importing prisma instance from lib/prisma.ts
-import { prisma } from '~/lib/prisma'
-
-  async function main() {
-    const posts = await prisma.post.findMany()
-    console.log(posts)
-  }
-  main()
-</script>
-```
-
-#### Using Prisma Client extensions 
-You can customize Prisma Client's capabilities by using [client extensions](https://www.prisma.io/docs/orm/prisma-client/client-extensions) in your `lib/prisma.ts` file. 
-Here is an example using [`prisma-extension-random`](https://github.com/nkeil/prisma-extension-random): 
-
-```ts
-// lib/prisma.ts 
-import { PrismaClient } from "@prisma/client"
-// Import extension after installing
-import prismaRandom from 'prisma-extension-random'
-
-const globalForPrisma = global as unknown as { prisma: PrismaClient }
-    
-// use .$extends method on PrismaClient()
-export const prisma = globalForPrisma.prisma || new PrismaClient().$extends(prismaRandom())
-    
-if (process.env.NODE_ENV !== 'production') globalForPrisma.prisma = prisma
-    
-export default prisma
-```
-
-Here is an example usage of your Prisma Client instance with the extension in your `.vue` file: 
-
-```vue
-<script lang="ts" setup>
-import { prisma } from '~/lib/prisma'
-  async function main() {
-    // Use findRandom() method from prisma-extension-random 
-    const posts = await prisma.post.findRandom() 
-    console.log(posts)
-  }
-  main()
-</script>
-```
-> To integrate [Prisma Pulse](https://www.prisma.io/docs/pulse/getting-started) or [Prisma Accelerate](https://www.prisma.io/docs/accelerate/getting-started) into your application, they must be configured as extensions using Option A's approach. 
-
-### Option B: `usePrismaClient()`
-This module exposes a [Nuxt composable](https://nuxt.com/docs/guide/directory-structure/composables) that is auto-imported inside your Vue files.
-
-This composable is using [Prisma Client](https://www.prisma.io/docs/orm/reference/prisma-client-reference) under the hood via a Nuxt plugin. It gives access to the Prisma Client in your Vue components. 
-
-```vue
-<script lang="ts" setup>
-  async function main() {
-    const prisma = usePrismaClient()
-    const posts = await prisma.post.findMany()
-    console.log(posts)
-  }
-  main()
-</script>
-```
+   ```ts
+   export default defineNuxtConfig({
+     modules: ["@prisma/nuxt"],
+   });
+   ```
 
+3. Start the development server:
 
-<!-- Badges -->
```

Comment:
We need these for the badges at the top of the readme. Please don't delete them 🙏

Reply:
Add that back in!